main/src/ca/uwaterloo/flix/util/ParOps.scala

```diff
@@ -43,6 +43,15 @@ object ParOps {
     result.seq
   }
 
+  /**
+    * Apply the given function `f` to each value in the map `m` in parallel.
+    */
+  @inline
+  def mapValues[K, A, B](m: Map[K, A])(f: A => B)(implicit flix: Flix): Map[K, B] =
+    parMap(m) {
```

Comment:
A question is whether using the parMap function here creates too many intermediate data structures.

Reply:
Maybe there is a nice general interface that all 3 functions can use.